[
    {
        "content": "<p>abrown <a href=\"https://github.com/bytecodealliance/wasmtime/pull/3062#issuecomment-874901006\">commented</a> on <a href=\"https://github.com/bytecodealliance/wasmtime/pull/3062\">issue #3062</a>:</p>\n<blockquote>\n<p>@afonso360, what do you make of <a href=\"https://github.com/bytecodealliance/wasmtime/pull/3062/checks?check_run_id=2988285291#step:12:1246\">this failure</a>?<br>\n</p>\n</blockquote>",
        "id": 245063079,
        "sender_full_name": "Wasmtime GitHub notifications bot",
        "timestamp": 1625588395
    },
    {
        "content": "<p>afonso360 <a href=\"https://github.com/bytecodealliance/wasmtime/pull/3062#issuecomment-874905450\">commented</a> on <a href=\"https://github.com/bytecodealliance/wasmtime/pull/3062\">issue #3062</a>:</p>\n<blockquote>\n<p>Hey, I think this is a bug in the aarch64 backend. I'll try to have a look at this today if possible.</p>\n</blockquote>",
        "id": 245063884,
        "sender_full_name": "Wasmtime GitHub notifications bot",
        "timestamp": 1625588769
    }
]